bin/make-release.sh

```diff
@@ -34,7 +34,10 @@ git rm -rf vendor
 rm -rf vendor
 rm -f composer.lock
 composer install || fail "composer install failed"
-find vendor/ -type f -name "*.php" \
+find vendor/ -type f -name "*" \
+ | grep -v '/docs/' \
+ | grep -v '/doc/' \
+ | grep -v '/.git*' \
```

Comment:
There may be other hidden files (like `.travis.yml` and `.php_cs`) which should not be staged. Just use `/.*` please.

Reply:
There's also sometimes a `phpunit*` file. Please don't stage them as well.